notebooks/abbash_notebook.ipynb

```diff
@@ -2,21 +2,118 @@
  "cells": [
```

Comment:
<p>The <code>lvm</code>  files are regular <em>tab</em> - delimited files, so you can import <code>file.lvm</code>  with</p><p> </p><p><code>data = np.loadtxt('file.lvm', delimiter='\t')</code>  </p><p> </p><p>This creates an N x 2 array where N is number of data points,  the first column is time data, and the second column is strain data. </p><p> </p><p><code>time = data[:,0]</code></p><p><code>strain = data[:,1]</code>  </p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cooperrc/uconn_numpy_tutorials/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>